torch/quantization/_equalize.py

```diff
@@ -0,0 +1,89 @@
+import torch
+import torch.nn as nn
+
+def channel_range(tensor, axis =0):
+    ''' finds the range of weights associated with a specific channel
+
+    '''
+    size_of_tensor_dim = len(tensor.size())
+    mins = tensor
+    maxs = tensor
+    for i in range(size_of_tensor_dim - 1, axis, -1):
```

Comment:
Is the first loop trying to initialize the tensor?

Reply:
What i'm trying to do is apply the torch.min/max on every dim except the dim specified by axis, didn't see any function already made for this, but the idea is given a n-dim tensor with size [a,b,c.... x], i wanted to apply the min function on all the dim after axis (first for loop) and then apply the min function on all the dim before axis (the second for loop)